direct/data/datasets.py

```diff
@@ -6,11 +6,13 @@
 import contextlib
 import logging
 import pathlib
+import re
 import sys
 import xml.etree.ElementTree as etree  # nosec
 from enum import Enum
 from typing import Any, Callable, Dict, List, Optional, Sequence, Tuple, Union
 
+import h5py
 import numpy as np
 from omegaconf import DictConfig
```

Comment:
You can run isort . to sort the imports according to PEP8. Recommended.

Reply:
This is sorted

Reply:
isort has been ran for all code. `import` goes first then `from...import..`